src/Controllers/Admin/UserSettingsController.php

```diff
@@ -99,7 +157,9 @@ public function settingsMenu(User $user): array
             $menu[url('/users/' . $user->id . '/certificates')] = [
                 'title' => 'settings.certificates',
                 'icon' => 'card-checklist',
-                'permission' => $this->isIfsgSupporter() ? null : 'user.ifsg.edit',
+                'permission' => ($this->isIfsgSupporter() || $this->isDriverLicenseSupporter())
+                    ? null
+                    : ('user.ifsg.edit' || 'user.drive.edit'),
```

Comment:
@xuwhite That does not work that way as its resolving to true and thus hides the menu point from shicos.